src/simpledi.js

```diff
@@ -98,17 +98,38 @@ SimpleDi.withNew = function(Constructor) {
   };
 };
 
+SimpleDi._withoutNew = function(factory) {
```

Comment:
I dislike the fact that we expose a "private" static function. Either we expose it without the underline or we keep it completely private by defining just a function in the module